### ISSUE TITLE
Arcade-extensions needs a CI story.

### ISSUE THREAD

**Issue by riarenas** [dotnet/arcade#13095]
- [ ] This issue is blocking 
- [X] This issue is causing unreasonable pain

https://github.com/dotnet/arcade-extensions is the repo where we have azure devops extensions needed by engineering services infrastructure. This repo doesn't have any validation or a deployment process. It should have both.

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [ ] Internal Infrastructure Improvements

**Comment by missymessa** [comment]
Might be able to do this with this functionality now: https://learn.microsoft.com/en-us/azure/devops/extend/publish/command-line?toc=%2Fazure%2Fdevops%2Fmarketplace-extensibility%2Ftoc.json&view=azure-devops

**Comment by garath** [comment]
From that page:

> Select All accessible organizations, which is the only value that works when publishing via CLI. If you select only one organization, it results in an error, even if the PAT is valid

We can't make PATs for all organizations. This might be why it hasn't been adopted.

**Comment by epananth** [comment]
@ilyas1974 will talk to @tkapin

**Comment by ilyas1974** [comment]
Per our offline discussion, this is a great issue for a new member of the team to work on so they can become familiar with our environment.

**Comment by tkapin** [comment]
Agree, there is no urgency with this but it's a nice opportunity to learn how to set up the CI for a repo.

@ali-turan7 I've assigned this to you, please have a look at that once you have time.

**Comment by missymessa** [comment]
This repo was moved to the internal repos and has a CI pipeline set up for it now. Closing.

**Comment by riarenas** [comment]
The CI pipeline in the repo still doesn't deploy anything and it just builds the repository. Reopening as this still needs to be done.
